src/gc/transaction_level_gc_manager.cpp

```diff
@@ -81,13 +84,13 @@ void TransactionLevelGCManager::Running(const int &thread_id) {
 }
 
 
-void TransactionLevelGCManager::RecycleTransaction(std::shared_ptr<GCSet> gc_set, const cid_t &timestamp) {
```

Comment:
You were reviewing the removed code. Originally we use timestamp to hash the context. But now I have changed it to `thread_id`. We no longer pass timestamp to the GC, because the GC only needs to know the `epoch_id` of a GC context.